### ISSUE TITLE
False Positive(deny_job_container_latest_image): Ignore image fields used variables

### ISSUE THREAD

**Issue by suzuki-shunsuke** [suzuki-shunsuke/ghalint#1306]
```yaml
    container:
      image: mirror.gcr.io/${{needs.list.outputs.image}}
```

```console
$ ghalint run
May  1 11:39:14.064 ERR the job violates policies
program=ghalint
version=1.5.5
workflow_file_path=.github/workflows/wc-test.yaml
policy_name=deny_job_container_latest_image
reference=https://github.com/suzuki-shunsuke/ghalint/blob/main/docs/policies/007.md
job_name=test-alpine
error="job container image should be <image name>:<tag>"
```
